src/frontend/Page/Docs.elm

```diff
@@ -398,10 +472,12 @@ viewSidebar model =
   div
     [ class "pkg-nav"
     ]
-    [ lazy4 viewReadmeLink model.author model.project model.version model.focus
-    , br [] []
-    , lazy4 viewBrowseSourceLink model.author model.project model.version model.latest
-    , h2 [] [ text "Module Docs" ]
+    [ ul []
```

Comment:
I seems better to use an `ul` instead of `br` to get exactly the same spacing as the modules list.